Tools/scripts/deepfreeze.py

```diff
@@ -22,6 +22,9 @@
 verbose = False
 identifiers, strings = get_identifiers_and_strings()
 
+# This must be kept in sync with opcode.py
+RESUME = 151
```

Comment:
Non blocking nit: Can we have some way to fail the test suite or something if this is not being kept in sync? This can be the source of a lot of weird stuff if people are not aware of.